pkg/server/server.go

```diff
@@ -862,8 +864,12 @@ func (s *ImmuServer) CreateDatabaseV2(ctx context.Context, req *schema.CreateDat
 	s.logDBOptions(db.GetName(), dbOpts)
 
 	err = s.startReplicationFor(db, dbOpts)
-	if err != nil && err != ErrReplicatorNotNeeded {
-		return nil, fmt.Errorf("%w: while starting replication", err)
+	if err != nil {
+		if err == ErrReplicatorNotNeeded {
+			s.Logger.Warningf("Replication for database '%s' not required.", db.GetName())
```

Comment:
it shouldn't be a warning. It's possible to use an external replicator.
A replicator may be initialized if the required data to connect with the master database are specified. Validation will return error if only partial information was provided, but otherwise it should just info that no replicator was initialized

Reply:
ack